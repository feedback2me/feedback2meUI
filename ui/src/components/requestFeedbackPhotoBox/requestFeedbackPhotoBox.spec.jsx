import React from 'react';
import { shallow } from 'enzyme';
import { RequestFeedbackPhotoBox } from '.';

describe('Given RequestFeedbackPhotoBox component', () => {
    describe('when rendering', () => {
        const onUpload = jest.fn();
        const wrapper = shallow(
            <RequestFeedbackPhotoBox onUpload={onUpload} />,
        );

        it('should match with snapshot', () => {
            expect(wrapper).toMatchSnapshot();
        });
    });
});
