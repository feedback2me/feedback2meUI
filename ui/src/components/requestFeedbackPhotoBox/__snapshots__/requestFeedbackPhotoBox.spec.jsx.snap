// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given RequestFeedbackPhotoBox component when rendering should match with snapshot 1`] = `
<div>
  <div>
    <Styled(Card)
      extra={
        <UploadButton
          onUpload={[MockFunction]}
        />
      }
      title="Uploaded photos"
    >
      <RequestFeedback />
    </Styled(Card)>
  </div>
</div>
`;
